frontend/src/pages/HomePage.tsx

```diff
@@ -18,10 +19,21 @@ export const HomePage = () => {
       console.log('logging in', response.idToken);
 
       // TODO: Make request to /userLoginSignup using response.idToken
-
-      // user variable is placeholder for data which will be returned from BE
-      const user = { name: 'John Smith', faculty: '' };
-      user.faculty ? navigate('/chats') : navigate('/update-user');
+      try {
+        const res = await http.post('/userLoginSignup', undefined, {
+          headers: {
+            Authorization: `Bearer ${response.idToken}`,
+          },
+        });
+        console.log(res.data);
+        res.data.faculty ? navigate('/chats') : navigate('/update-user');
+      } catch (err: any) {
+        if (err.response) {
+          console.log(err.response.data);
+        } else {
+          console.log(`Error: ${err.message}`);
+        }
+      }
```

Comment:
If there is an error, rather than log it to the console, we should probably display it to the user. Maybe we can use a [Snackbar](https://mui.com/material-ui/react-snackbar/#simple-snackbars) that says the error message, or if there is no error, something like "An error occurred. Please try again later."